activiti-core/activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/process/ProcessRuntimeIT.java

```diff
@@ -268,6 +274,46 @@ public void createProcessInstanceAndValidateHappyPath() {
         assertThat(RuntimeTestConfiguration.discardImageConnectorExecuted).isEqualTo(false);
     }
 
+    @Test
+    public void createLinkEventProcessInstanceAndValidateHappyPath() {
+        //when
+        ProcessInstance linkProcess = processRuntime.start(ProcessPayloadBuilder.start()
+            .withProcessDefinitionKey(LINK_PROCESS)
+            .withVariable("expectedKey",
+                true)
```

Comment:
You don't need to provide variables here, as they are not going to be used by the test.